src/main/resources/config/application-disruptor.yml

```
@@ -1,22 +1,30 @@
       # Tune the connection timeout to avoid long waits
-      connection-timeout: 30000
+      connection-timeout: 20000
       # Enable leak detection to find potential connection leaks
-      leak-detection-threshold: 60000
+      leak-detection-threshold: 30000
       # Set housekeeping period to higher value to reduce frequency
-      housekeeping-period-ms: 60000 
\ No newline at end of file
+      housekeeping-period-ms: 30000
+      # Enable metrics for monitoring
+      metrics-registry-type: micrometer 
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)